packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.renovaterc.json

```diff
@@ -39,8 +39,8 @@
       ],
       "postUpgradeTasks": {
         "commands": [
-          "yarn install",
-          "yarn run swagger:generate"
+          "<%=packageManager%> install",
+          "<%=packageManager%> exec swagger:generate"
```

Comment:
it is not a `run` instead of exec.
`swagger:generate` does not exist in package.json. I think it's swagger:regen

Reply:
From what I see in the package.json it seems to be `generate` only